packages/app-builder-lib/src/electron/electronMac.ts

```diff
@@ -214,7 +214,7 @@ export async function createMacApp(packager: MacPackager, appOutDir: string, asa
   }
 
   if (asarIntegrity != null) {
-    appPlist.AsarIntegrity = JSON.stringify(asarIntegrity)
+    appPlist.ElectronAsarIntegrity = asarIntegrity
```

Comment:
Will this break anything for electron pre-15?

Reply:
There is no mention of `AsarIntegrity` in electron `14-x-y`, `13-x-y`, `12-x-y`, and `11-x-y` so it is safe to say that it wasn't used at all.